debian/k8s-cni/99_bridge.conf

```diff
@@ -0,0 +1,18 @@
+{
+  "cniVersion": "0.1.0",
+  "name": "kubenet",
```

Comment:
Does kubenet only work on GCE?
Is there a convenient way to support other network drivers?